src/transformers/generation/stopping_criteria.py

```diff
@@ -98,7 +98,7 @@ class MaxNewTokensCriteria(StoppingCriteria):
 
     def __init__(self, start_length: int, max_new_tokens: int):
         warnings.warn(
-            "The class `MaxNewTokensCriteria` is deprecated. "
+            "The class `MaxNewTokensCriteria` is deprecated and will be removed in v4.42. "
```

Comment:
nit - this will be in the 4.41 release and we normally deprecate in 2 release cycles

```suggestion
            "The class `MaxNewTokensCriteria` is deprecated and will be removed in v4.43. "
```

Reply:
Added :) 

My intuition for the new version + 1 was due to how old this one was: it was deprecated 3 years ago